packages/sdk-js/package.json

```diff
@@ -39,10 +39,12 @@
     "webpack-cli": "^4.9.2"
   },
   "dependencies": {
+    "@kiltprotocol/augment-api": "1.11210.0-rc",
```

Comment:
So no we're pulling a specific version RC into `develop`? Would we want to have `master` being merged here, and then pull from a specific branch upon a release candidate?

Reply:
57d08f9f0f6dc0230f3cd93f6f52e776751cc878

Reply:
We're still pinning it to a specific version from our `develop` branch? Wouldn't `latest` be enough here?

Reply:
nope, because this is not latest

Reply:
> Would we want to have `master` being merged here, and then pull from a specific branch upon a release candidate?

I'm not sure what you're referring to with merging master - this is not going to happen as master is still on pre-V1. But before releasing V1, this would certainly be replaced with a properly released version of the augment-api & type-definitions packages

Reply:
No I mean that this `develop` branch would simply track the `master` branch of the new repo. Then each release would track a specific version of the new repo. So we can make sure that any changes on `master` in the new repo are compatible with the `develop` version of this, kinda like we do for integration tests against the chain `develop` branch.

Reply:
I agree that releasing will involve pinning the version, but before then I think it's a bit "weird" to have `develop` being pinned to a specific released version of the new augmentation package.

Reply:
a) latest is a tag. this will select one specific version tagged with latest, which is the latest full release and not the rc that I'm currently experimenting with.
b) what's the point of a peer dependency if you then require one specific version?

Reply:
Sorry, I just realized this was changed to be a peer dependency. 

> a) latest is a tag. this will select one specific version tagged with latest, which is the latest full release and not the rc that I'm currently experimenting with.

Is this supposed to change then at some point? Is it just temporarily used to verify that things work?

Reply:
yes, absolutely. Once we fully release the augment-api package we can switch this to a sensible version range, such as `^1.11210.0`, which is essentially equivalent to `latest` (allows all higher versions accept new majors).